src/components/Amount.vue

```diff
@@ -10,10 +10,18 @@ import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
 import { FormattableNumber } from '@nimiq/utils';
 type BigInteger = import ('big-integer').BigInteger;
 
+export function amountValidator(value: unknown): boolean {
+    return typeof value === 'number' || typeof value === 'bigint'
+        || (value && value.constructor && value.constructor.name.endsWith('Integer'));
+}
```

Comment:
Since this is not validating the amount itself but only its type, I'd call this `amountTypeValidator`, or `isNumberValidator`.

Reply:
`isNumberValidator` would be missleading, as it does not only check for type `number`, i.e. someone calling this method couldn't make the assumption that the value is safe to use as a number. `amountTypeValidator` would be possible but I see `amountValidator` a good fit as it validates the `amount` prop for the `Amount` component. Whether it only validates the type or enforces further restrictions is of less interest.